include/engine/routing_algorithms/routing_base.hpp

```diff
@@ -484,6 +484,69 @@ template <class DataFacadeT, class Derived> class BasicRoutingInterface
         unpacked_path.emplace_back(t);
     }
 
+    /**
+     * A duplicate of the above `UnpackEdge` function, but returning full EdgeData
+     * objects for the unpacked path. Used in the tile plugin to find outgoing
+     * edges from a given turn.
+     */
+
```

Comment:
`FindSmallestEdge` isn't quite the same, but I'm not clear on whether the difference is important:

UnpackPath uses:

```
        if ((facade->GetTarget(edge_id) == edge.second) && (weight < edge_weight) &&
            facade->GetEdgeData(edge_id).forward)
        {
            smaller_edge_id = edge_id;
            edge_weight = weight;
        }
```

whereas `FindSmallestEdge` uses:

```
    if (target == to && weight < smallest_weight)
    {
        smallest_edge = edge;
        smallest_weight = weight;
    }
```